epidose/device/beacon_tx_unlinkable_d.py

```diff
@@ -84,6 +85,17 @@ def set_transmit(interface, ephid, rssi):
     run_command(cmd)
 
 
+def generate_random_bdaddr():
+    bdaddr = secrets.token_hex(6)
```

Comment:
Would `mac_addr` be more appropriate?

Reply:
MAC address is used for both Ethernet and Bluetooth technologies.
I have selected bdaddr because it stands for the Bluetooth device address.
Maybe, we can also use bd_addr, but if you insist we can change it to mac_addr

Reply:
OK.  Then add a function comment, documenting what the function does and explaining that it returns a random "Bluetooth Device (MAC) Address  (bdaddr)".